### ISSUE TITLE
Show note attachments

### ISSUE THREAD

**Issue by jclement** [jclement/mattermost-dynamics-bot#3]
1. Show attachments when present on note.
2. Separate API to fetch attachment and download with appropriate mimetype / name

See "mimetype" and "documentbody_binary" and "filename" fields on annotation.

**Comment by ctrowat** [comment]
I took a stab at this - I threw in 2 different ways to download the attachment, one opening it in a new tab and one using an iframe.  Maybe there's an even better way to do this than what I've put in there?
I don't like the new tab open-then-close behaviour so I prefer the iframe for that reason but I have a deeply ingrained hate for iframes so feel free to just gut them both if you've got something better

The URL for downloading the attachment is not something I'm at all attached to but the nice thing about throwing the filename on the end of the URL is it makes the browser save the file with the correct name.

**Comment by jclement** [comment]
I think either works.

Apparently we're not supposed to use that type of attachment any more (we're supposed to use that folder).  I've implemented the folder type attachments.  I think this style is read-only for historical use only.

**Comment by ctrowat** [comment]
Yeah - we were supposed to stop using them but just in case you need to load a case where they have been previously used I figured they'd be nice to have on there.
